SETUP.md

```diff
@@ -188,11 +188,87 @@ packages/files, and thus probably don't use the debian image.
 
 ## Clean-up
 
-If you'd like to delete the nodes in your database, you execute the query:
+To delete the nodes in your database, you execute the query:
 ```
 MATCH (n) DETACH DELETE n;
 ```
 
+# Example 3: OSV Certifier
+
+Perform a [cleanup] to remove all nodes and start from scratch.
+
+In this example, we will ingest an SPDX SBOM for a custom vulnerable image that contains `log4j` and `tesx4shell`.
+
+Running the OSV Certifier will allow all the packages to be evaluated against the OSV database. If a vulnerability
+is found, a vulnerability node will be generated containing the OSV ID that can be queried further for more information.
+Along with the vulnerability node, a vulnerability attestation node is also generated based on a custom predicate defined
+in https://github.com/guacsec/guac/blob/main/pkg/certifier/attestation/attestation_vuln.go and an example defined in 
```

Comment:
nit: can we use the short version `pkg/certifier/attestation/attestation_vuln.go` and then link to the full URL instead? for readability!

Reply:
oops. I will create a small PR to add this in.